### PR TITLE
LPS-200398 DDMStructures selector in WC Admin view

### PR DESCRIPTION
**Pull request by jkappler**
Step to test this:
1. Enable: `LPS-194763` and `LPS-197692` feature flags.
2. Deploy everything.
3. Create a couple of Web Content structures.
4. Go to Web Content admin and select some of them:

[Screencast from 2023-11-10 13-28-32.webm](https://github.com/liferay-echo/liferay-portal/assets/800645/bb1c71e5-594b-4b78-a3ef-5d8c9eee4b7c)

@p2kmgcl

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
Niiice :clap:

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/configuration_browse/LabelList.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	onItemsChange: (nextItems: Item[]) => void;
+}
+
+export function LabelList({items, onItemsChange}: Props) {
+	const onRemoveButtonClick = (item: Item) => {
+		onItemsChange(items.filter(({value}) => value !== item.value));
+	};
+
+	return (
+		<div className="form-control form-control-tag-group" role="list">
```
[the remark this answers is not in the document]
Comment (p2kmgcl) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/configuration_browse/LabelList.tsx:26]: I tried, and it showed bullet items, something like this :cry: 
`* item A * item B * item C`

I tried to remove them with some CSS class and couldn't find any, and I didn't want to add custom CSS just for that.

Please tell me if you know how to do that :sweat_smile:

**Review by lfbesada — APPROVED** [review]
Very nice! :heart_eyes:

**Comment by lfbesada** [comment]
ci:forward

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/configuration_browse/LabelList.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/configuration_browse/LabelList.tsx:26]: I think we have `list-unstyled` for that

[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/configuration_browse/LabelList.tsx:26]: But I don't think this matters too much

[the remark this answers is not in the document]
Comment (p2kmgcl) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/configuration_browse/LabelList.tsx:26]: It works! I am updating that. Thanks!!!! :smi

**Comment by p2kmgcl** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
https://github.com/liferay-echo/liferay-portal/pull/14328

**Pull request closed without merging**